packages/ckeditor5-ckbox/tests/ckboximageedit/ckboximageeditcommand.js

```diff
@@ -0,0 +1,231 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/* global window, btoa */
+
+import { global } from '@ckeditor/ckeditor5-utils';
+import { Command } from 'ckeditor5/src/core';
+import ClassicTestEditor from '@ckeditor/ckeditor5-core/tests/_utils/classictesteditor';
+import { Paragraph } from '@ckeditor/ckeditor5-paragraph';
+import { Heading } from '@ckeditor/ckeditor5-heading';
+import { Essentials } from '@ckeditor/ckeditor5-essentials';
+import CloudServices from '@ckeditor/ckeditor5-cloud-services/src/cloudservices';
+import testUtils from '@ckeditor/ckeditor5-core/tests/_utils/utils';
+import CloudServicesCoreMock from '../_utils/cloudservicescoremock';
+import TokenMock from '@ckeditor/ckeditor5-cloud-services/tests/_utils/tokenmock';
+
+import CKBoxImageEditCommand from '../../src/ckboximageedit/ckboximageeditcommand';
+
+describe( 'CKBoxImageEditCommand', () => {
+	let editor, domElement, command;
+
+	beforeEach( async () => {
+		TokenMock.initialToken = [
+			// Header.
+			'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9',
+			// Payload.
+			btoa( JSON.stringify( { auth: { ckbox: { workspaces: [ 'workspace1' ] } } } ) ),
+			// Signature.
+			'signature'
+		].join( '.' );
+
+		domElement = global.document.createElement( 'div' );
+		global.document.body.appendChild( domElement );
+
+		window.CKBox = {
+			mountImageEditor: sinon.stub()
```

Comment:
This looks like not been restored. I'd suggest using testUtils.createSinonSandbox(); at the top level describe block.

Reply:
Done.